signer/src/storage/in_memory.rs

```diff
@@ -1284,24 +1284,20 @@ impl super::DbWrite for SharedStore {
     {
         let mut store = self.lock().await;
         if let Some((_, shares)) = store.encrypted_dkg_shares.get_mut(&aggregate_key.into()) {
```

Comment:
maybe nit: should we add also for the in memory the constraint of not updating if the state is not pending?

Reply:
Ah, yeah we should.

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/1316/commits/97142e3f25098c00851959ea9429a4c79556ae3e